auditors/nonroot/nonroot.go

```diff
@@ -45,11 +49,46 @@ func (a *RunAsNonRoot) Audit(resource k8stypes.Resource, _ []k8stypes.Resource)
 }
 
 func auditContainer(container *k8stypes.ContainerV1, resource k8stypes.Resource) *kubeaudit.AuditResult {
+	if !isContainerRunAsUserNil(container) {
+		if *container.SecurityContext.RunAsUser == 0 {
+			return &kubeaudit.AuditResult{
+				Name:     RunAsUserCSCRoot,
+				Severity: kubeaudit.Error,
+				Message:  "runAsUser is set to UID 0 (root user) in the container SecurityContext. It should be set to a value > 0.",
+				Metadata: kubeaudit.Metadata{
+					"Container": container.Name,
+				},
+			}
+		}
+
+		return nil
+	}
+
+	podSpec := k8s.GetPodSpec(resource)
+	if podSpec == nil {
+		return nil
+	}
+
```

Comment:
This is just an idea. As I was running the tests ("run-as-user-psc-0-csc-1.yml") I missed seeing some info/warning about the Pod Security Context with `runAsUser` set to 0. So I thought that we could change the code a bit so that we would get an error at an info level just to let the user know that runAsUser set to 0 is not recommended. If we change it, though, we will need to modify the tests a bit. Not sure it's worth it. What do you think?

```suggestion
	podSpec := k8s.GetPodSpec(resource)
	if podSpec == nil {
		return nil
	}

	if !isContainerRunAsUserNil(container) {
		if *container.SecurityContext.RunAsUser == 0 {
			return &kubeaudit.AuditResult{
				Name:     RunAsUserCSCRoot,
				Severity: kubeaudit.Error,
				Message:  "runAsUser is set to UID 0 (root user) in the container SecurityContext. It should be set to a value > 0.",
				Metadata: kubeaudit.Metadata{
					"Container": container.Name,
				},
			}
		} else {
			if !isPodRunAsUserNil(podSpec) {
				if *podSpec.SecurityContext.RunAsUser == 0 {
					return &kubeaudit.AuditResult{
						Name:     RunAsUserPSCRoot,
						Severity: kubeaudit.Info,
						Message:  "runAsUser is set to 0 in the PodSecurityContext. It should be set to a value > 0 in either the PodSecurityContext or in the container SecurityContext.",
						Metadata: kubeaudit.Metadata{
							"Container": container.Name,
						},
					}
				}
			}
		}
	}
```